Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org' do
   gem 'rake'
-  gem 'cocoapods', '1.7.5'
+  gem 'cocoapods', '1.6.1'
```

Comment:
Was this downgrade intentional?

Reply:
Yep. We are on 1.6 on other apps, so I thought it would be nice to sync.

Reply:
I think 1.7.x is required to work correctly with Xcode 11. Is that right @jkmassel?